config/config/common-config.yaml

```diff
@@ -548,6 +548,9 @@ ingressNginx:
     nodeSelector: {}
     enablepublishService: false
 
+    ## Enable validation of Nginx annotations on Ingress resources
```

Comment:
maybe, also, add the reason we kept it to false (something very short). Just to know that we want to revisit this in the future